controller/api/models.py

```diff
@@ -298,6 +298,24 @@ def _start_containers(self, to_add):
         if set([c.state for c in to_add]) != set(['up']):
             err = 'warning, some containers failed to start'
             log_event(self, err, logging.WARNING)
+        # if the user specified a health check, try checking to see if it's running
+        try:
```

Comment:
I think setting a [request timeout](http://requests.readthedocs.org/en/latest/user/quickstart/#timeouts) of 60 seconds [a la Heroku](https://devcenter.heroku.com/articles/error-codes#r10-boot-timeout) would make sense. I don't believe we should add some retry logic though as the first response should be enough to determine if the app is responding as expected.